addons/syscalls/syscallinfo.py

```diff
@@ -33,23 +33,23 @@ def parse_table(self, filename):
 
         for line in lines[1:]:
             parts = line.split("\t")
-            self.entries[parts[1]] = self.getEntryDict(
+            self.entries[parts[1]] = self.get_entry_dict(
                 line.split("\t"), identifiers)
 
-    def getEntryByID(self, idx):
+    def get_entry_by_id(self, idx):
         for entry in self.entries:
             if self.entries[entry]["#"] == str(idx):
                 return self.entries[entry]
 
         return None
 
-    def getEntryByName(self, name):
+    def get_entry_by_name(self, name):
         if name in self.entries:
```

Comment:
we could replace this with one liner `return self.entries.get(name)` which returns `None` if the entry doesn't exist.